### ISSUE TITLE
Searching through the source code

### ISSUE THREAD

**Issue by Rob--W** [mdamien/chrome-extensions-archive#2]
Could you index extensions by manifest keys (e.g.  `permissions` / `optional_permissions`, `content_scripts`), and add the ability to search through the source code of extensions that match the query?

For example, I'd like to know how many extensions use the webRequest API in a blocking fashion (=webRequest + webRequestBlocking permissions) and block "ping" requests (to make informed decisions on https://crbug.com/611453).

**Comment by mdamien** [comment]
Yes, that's really the next step (full source code search + metadata search).

I'm currently indexing extensions infos + metadata to elasticsearch, source code search should come soon after that.

**Comment by mdamien** [comment]
_Reminder to myself: just extract the archives and do a normal recursive file search 
http://beyondgrep.com/more-tools/ for efficient search_

**Comment by mdamien** [comment]
Here's a very basic prototype using `ag`: [http://codesearch.x.dam.io/?q=your_thing](http://codesearch.x.dam.io/?q=your_thing)

EDIT: now switching to another strategy and doing it with postgres instead
